src/core/MythosArea.ttslua

```diff
@@ -108,6 +134,7 @@ function onCollisionExit(collisionInfo)
     tokenData = {}
```

Comment:
Now that you have a reference to the card, you should check if the exiting object is the scenario reference card